src/main/kotlin/org/octopusden/octopus/automation/teamcity/TeamcityCreateBuildChainCommand.kt

```diff
@@ -138,6 +138,7 @@ class TeamcityCreateBuildChainCommand : CliktCommand(name = COMMAND) {
         setProjectParameter(project.id, "COMPONENT_NAME", componentName)
         setProjectParameter(project.id, "RELENG_SKIP", "false")
         setProjectParameter(project.id, "PROJECT_VERSION", minorVersion)
+        setProjectParameter(project.id, "JDK_VERSION", component.buildParameters?.javaVersion ?: "1.8")
```

Comment:
1. Check "If a component uses non-default JDK version"
2. Don't hard-code any default version, just not set parameter